### PR TITLE
WIP: Py cdp

### PR DESCRIPTION
**Pull request by AutomatedTester**
WIP for adding CDP to python

### CODE REVIEW

File: common/devtools/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,10 @@
+filegroup(
+    name="devtools-json",
+    srcs=[
+        "browser_protocol.json",
+        "js_protocol.json",
+    ],
+    visibility=[
+        "//java/client/src/org/openqa/selenium/devtools"
```
Comment (shs96c) [inline common/devtools/BUILD.bazel]: This needs to be set to `//java/client/src/org/openqa/selenium/devtools:__pkg__` This is going to cause a circular dep

Comment (AutomatedTester) [reply common/devtools/BUILD.bazel]: fixed

File: py/selenium/webdriver/support/cdp.py [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+class InternalError(Exception):
+    ''' This exception is only raised when there is faulty logic in TrioCDP or
+    the integration with PyCDP. '''
+
+
+class CDPBase:
+
+    def __init__(self, ws, session_id, target_id):
+        self.ws = ws
+        self .session_id = session_id
```
Comment (raju249) [inline py/selenium/webdriver/support/cdp.py:141]:
```suggestion
        self.session_id = session_id
```

**Comment by AutomatedTester** [comment]
merged into trunk so closing

**Pull request closed without merging**